### PR TITLE
Adding Pandemic by A.G. Riddle

### PR DESCRIPTION
**Pull request by Rogue-System**
First of a two part series.  Took the first section of the goodreads description for the sake of brevity.  It's several paragraphs long.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -1015,6 +1015,18 @@ If you liked Cloud Atlas this is a good read.
 
 _Novels which focus on the near-future unintended consequences of biotechnology revolutions._
 
+#### [Pandemic](https://www.goodreads.com/book/show/34811896-pandemic) (2017) *by [A.G. Riddle](https://www.goodreads.com/author/show/3087199.A_G_Riddle)
+
+The first book in a two part series called the extinction files, the story explores a bioterrorist plot attempting to usurp the major world governments, with an extra scifi twist. [@Rogue-System](https://github.com/Rogue-System)
```
Comment (RichardLitt) [inline readme.md]: What is your opinion about this book and series? 

Also, add a dash between the period and your name.

File: readme.md [hunk 2]
```diff
@@ -1015,6 +1015,18 @@ If you liked Cloud Atlas this is a good read.
 
 _Novels which focus on the near-future unintended consequences of biotechnology revolutions._
 
+#### [Pandemic](https://www.goodreads.com/book/show/34811896-pandemic) (2017) *by [A.G. Riddle](https://www.goodreads.com/author/show/3087199.A_G_Riddle)
```
Comment (RichardLitt) [inline readme.md]: Bold block needs to be closed.

Comment (Rogue-System) [reply readme.md]: Sorry, been a bit busy, I'll get it corrected tonight when I get home.

Comment (Rogue-System) [reply readme.md]: I think I figured it out, should be fixed now.

**Comment by RichardLitt** [comment]
Thanks! You got it. :)

**Pull request merged**